plugins/plugin-java/che-plugin-java-plain/che-plugin-java-plain-server/src/main/resources/project

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I've noticed you're not using jdt anymore to generate classpath/project files. What's the reason?

Reply:
The main reasons are:  
1/After project creation we don't have full structure of the project when JDT.LS isn't activated (wasn't added to installers)
2/Then if workspace has bad projects (without .classpath and .project) jdt.ls woun't recognize these projects as plain java projects.